docs/build-integration.md

```diff
@@ -0,0 +1,37 @@
+---
+id: build-integration
+title: Polkadot Integration Guide
+sidebar_label: Getting Started
+---
+
+Welcome to the Polkadot integration guide. This guide will provide all the information you need to
+get started. It targets wallets and custodians, but will also be useful to any infrastructure
+provider such as a validator or node operator.
+
+# Structure
+
+The guide focuses on Polkadot and Kusama, but the principles will apply to parachains and other
+Substrate-based chains. The guide contains four sections:
+
+1. [Protocol Info](build-protocol): The entry point to the guide. Please read it carefully as it
+   provides information about Polkadot that differentiates it from other blockchains. Use this page
+   to check your assumptions.
+1. [Node Management](build-node-management): This page will guide you to starting and monitoring a
```

Comment:
```suggestion
2. [Node Management](build-node-management): This page will guide you to starting and monitoring a
```

Reply:
This is normal markdown so that when you edit to add another list item later, you don't need to redo everything. Pull this branch, run `yarn start`, and see what happens :).